src/Microsoft.ML.Data/DataLoadSave/Database/DatabaseSource.cs

```diff
@@ -10,21 +10,37 @@ namespace Microsoft.ML.Data
     /// <summary>Exposes the data required for opening a database for reading.</summary>
     public sealed class DatabaseSource
     {
+        public static readonly int DefaultCommandTimeoutInSeconds = 30;
```

Comment:
nit: does this need to be `public`?
Also, should it be `const` instead of `readonly`? Since we aren't modifying it in the constructor